pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -589,11 +589,16 @@ func init() {
 					Verbs:     sets.NewString("list", "watch", "get"),
 					Resources: sets.NewString("storageclasses"),
 				},
-				// Gluster provisioner
+				// Glusterfs provisioner
 				{
 					Verbs:     sets.NewString("get", "create", "delete"),
 					Resources: sets.NewString("services", "endpoints"),
 				},
+				// Glusterfs & Ceph provisioner
+				{
+					Verbs:     sets.NewString("get"),
+					Resources: sets.NewString("secrets"),
```

Comment:
can you pick 35675 as part of this, and keep an eye on new PV provisioners upstream to make sure they follow that restriction?